srcopsmetrics/processing.py

```diff
@@ -328,7 +324,7 @@ def _extract_review_data(
 
     @staticmethod
     def _evaluate_reviewer_data(
-        pr: Dict[str, Any], reviewer: str, review_submission_dt: datetime.timestamp, extracted_data: Dict[str, Any]
+        pr: Dict[str, Any], reviewer: str, review_submission_dt, extracted_data: Dict[str, Any]
```

Comment:
What is the typeof `review_submission_dt`

Reply:
type hint added

Reply:
type hint corrected